dcli/language/cpp.py

```diff
@@ -11,16 +21,41 @@
 
 TARGET_FILE = 'Target'
 
+
 class Cpp(Language):
     def __init__(self):
         Language.__init__(self,
-            'cpp',
-            TEMPLATE_PATH,
-            SUCCESS_PATH,
-            APP_PATH,
-            SOLVE_PATH,
-            RUN_PATH,
-            TARGET_FILE,
-            'cpp',
-            ASSET_PATHS
-        )
+                          'cpp',
+                          TEMPLATE_PATH,
+                          SUCCESS_PATH,
+                          APP_PATH,
+                          SOLVE_PATH,
+                          RUN_PATH,
+                          TARGET_FILE,
+                          'cpp',
+                          ASSET_PATHS
+                          )
+
+    def format_data(self, datatype_holder):
+        ret = ''
```

Comment:
what does `ret` means ? You can find something more explicit.  
Same for the other languages

Reply:
ret is a nickname for return variable